content/en/docs/tasks/security/authorization/authz-ingress/index.md

```diff
@@ -31,143 +26,390 @@ through the Istio ingress gateway with this command:
     $ kubectl apply -f <(istioctl kube-inject -f @samples/httpbin/httpbin-gateway.yaml@) -n foo
     {{< /text >}}
 
-* See [Source IP for Services with `Type=NodePort`](https://kubernetes.io/docs/tutorials/services/source-ip/#source-ip-for-services-with-type-nodeport)
-for more information. Update the ingress gateway to set `externalTrafficPolicy: local` to preserve the
-original client source IP on the ingress gateway using the following command:
+* Turn on RBAC debugging in Envoy for the ingress gateway:
 
     {{< text bash >}}
-    $ kubectl patch svc istio-ingressgateway -n istio-system -p '{"spec":{"externalTrafficPolicy":"Local"}}'
+    $ kubectl get pods -n istio-system | grep ingress | awk '{print $1}' | while read -r pod; do istioctl proxy-config log "$pod" -n istio-system --level rbac:debug; done
     {{< /text >}}
 
-*  Follow the instructions in
-    [Determining the ingress IP and ports](/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports)
-    to define the `INGRESS_HOST` and `INGRESS_PORT` environment variables.
-
 * Verify that the `httpbin` workload and ingress gateway are working as expected using this command:
 
     {{< text bash >}}
     $ curl "$INGRESS_HOST":"$INGRESS_PORT"/headers -s -o /dev/null -w "%{http_code}\n"
```

Comment:
A best practice in bash is to put quotes around the whole arg, not each variable:
```suggestion
    $ curl "$INGRESS_HOST:$INGRESS_PORT/headers" -s -o /dev/null -w "%{http_code}\n"
```